geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreQueryTest.scala

```diff
@@ -479,14 +479,14 @@ class AccumuloDataStoreQueryTest extends Specification with TestWithMultipleSfts
     }
 
     "kill queries after a configurable timeout" in {
-      import scala.concurrent.duration._
-
+      skipped("relies on thread.sleep timing")
```

Comment:
Hmm....  Anything we can do for this test?

Reply:
nothing i can think of... we can't use the 'eventually' construct b/c we only throw exceptions on 'next' and that's not a idempotent call. or we could enable the test, but i feel like the CI build will fail randomly on it if we do.